source/simulator/core.cc

```diff
@@ -113,15 +113,15 @@ namespace aspect
      * computation.
      */
     template <int dim>
-    std_cxx11::unique_ptr<Mapping<dim> > construct_mapping(const GeometryModel::Interface<dim> &geometry_model,
-                                                           const InitialTopographyModel::Interface<dim> &initial_topography_model)
+    Mapping<dim> *construct_mapping(const GeometryModel::Interface<dim> &geometry_model,
+                                    const InitialTopographyModel::Interface<dim> &initial_topography_model)
```

Comment:
Yeah. The problem is that ``scoped_ptr`` is not a drop-in replacement for ``unique_ptr``. We could have switched to a ``shared_ptr`` here, I guess.